composites/Plugin/Shared/components/GutenbergToggle.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from "react";
+import classnames from "classnames";
+import noop from "lodash/noop";
+
+function GutenbergToggle( { className, checked, id, onChange = noop, ...props } ) {
+	const wrapperClasses = classnames(
+		'components-form-toggle',
+		className,
+		{ 'is-checked': checked },
+	);
+
```

Comment:
so to my understanding, this gets properly styled only when in Gutenberg. When used outside Gutenberg, it will be completely unstyled (as in the standalone app example > "Keyword" tab). Is this what we want?